src/swtpm_setup/swtpm_setup.c

```diff
@@ -1590,5 +1602,9 @@ int main(int argc, char *argv[])
     g_strfreev(swtpm_prg_l);
     g_free(gl_LOGFILE);
 
+    if (certsdir && g_rmdir(certsdir) != 0)
+        logerr(gl_LOGFILE, "Could not remove temporal directory for certs: %s\n",
```

Comment:
same here. will merge then.

Reply:
Thanks will fix them and commit message as well.